src/HttpCommon.jl

```diff
@@ -34,8 +35,8 @@ It has five fields:
 * `data`: the request data as a vector of bytes
 """
 type Request
-    method::UTF8String      # HTTP method string (e.g. "GET")
```

Comment:
Like I mentioned in the `BinDeps` PR I think it is better to not change the field type on 0.4 unless there's clear evidence that no one is depending on the type of it.